_posts/2020-03-05-git-secret.md

```diff
@@ -12,51 +12,52 @@ git-secret - bash tool to store private data inside a git repo.
 
 These steps cover the basic process of using `git-secret`:
 
-0. Before starting, [make sure you have created `gpg` RSA key-pair](#using-gpg): public and secret key identified by your email address.
+0. Before starting, [make sure you have created a `gpg` RSA key-pair](#using-gpg): a public and a secret key identified by your email address.
 
 1. Begin with an existing or new git repository. You'll use the 'git secret' commands to add the keyrings and information
-to make the git-secret hide and reveal files in this repository.
+to make `git-secret` hide and reveal files in this repository.
 
-2. Initialize the `git-secret` repository by running `git secret init` command. the `.gitsecret/` folder will be created,
-**Note** all the contents of the `.gitsecret/` folder should be checked in, /except/ the `random_seed` file.
-In other words, of the files in .gitsecret, only the random_seed file should be mentioned in your .gitignore file.
+2. Initialize the `git-secret` repository by running `git secret init` command. The `.gitsecret/` folder will be created.
+**Note** all the contents of the `.gitsecret/` folder should be checked in, **/except/** the `random_seed` file.
+In other words, of all the files in `.gitsecret/`, only the `random_seed` file should be mentioned in your `.gitignore` file.
+By default, `git secret init` will add the file `.gitsecret/keys/random_seed` to your `.gitignore` file.
 
-3. Add the first user to the git-secret repo keyring by running `git secret tell your@gpg.email`.
+3. Add the first user to the `git-secret` repo keyring by running `git secret tell your@gpg.email`.
 
 4. Now it's time to add files you wish to encrypt inside the `git-secret` repository.
-It can be done by running `git secret add <filenames...>` command. Make sure these files are ignored by mentions in
-.gitignore, otherwise `git-secret` won't allow you to add them, as these files could be stored unencrypted.
+This can be done by running `git secret add <filenames...>` command. Make sure these files are ignored by mentions in
+`.gitignore`, otherwise `git-secret` won't allow you to add them, as these files could be stored unencrypted. Actually, in the default configuration it will automatically add the files to `.gitignore` for you.
```

Comment:
Please replace the last sentence with:

"In the default configuration, `git-secret add` will automatically add the unencrypted versions of the files to `.gitignore` for you."